libs/etterlatte-regler/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
     kotlin("jvm")
 }
 
-version = "0.1-BETA"
+version = "0.1"
```

Comment:
Skulle vi kanskje ha startet på versjon `1.0` for ordens skyld? :D

Reply:
Sikkert en god ide 👍